README.md

```diff
@@ -19,31 +19,31 @@ Create a `.env` file with environment variables. You can use the `.env.example`
 
 Here's the list of all the environment variables:
 
-| Env variable                                           | Description
```

Comment:
This diff was some autoformatting by my VS Code.. If we do not want this I have to revert and Save without formatting.